.github/FUNDING.yml

```diff
@@ -1,2 +1,2 @@
 open_collective: osca
-
+github: oscafrica
```

Comment:
The GH Sponsors slug is [oscafrica](https://github.com/sponsors/oscafrica). All official usernames are `oscafrica` (for more clarity). OC was created earlier, hence the use of `osca`, and since it's longer, we didn't consider updating.

Reply:
Since we are going with oscafrica, we can go ahead and merge this.